src/main/java/org/takes/facets/auth/codecs/CcAES.java

```diff
@@ -145,15 +123,42 @@ public Identity decode(final byte[] bytes) throws IOException {
     }
 
     /**
+     * This method will create AlgorithmParameterSpec with the block size.
+     */
+    private void initAlgorithmParameterSpec() {
```

Comment:
@sebing make this method returning `AlgorithmParameterSpec` and call it from constructor